### DISCUSSION TITLE
LIBS variable being cleared in makefile

### DISCUSSION THREAD

**Discussion by malinastrand** [micropython/micropython#14101] [nRF]
I am writing a C module to use with the nrf port. I want to add my own libraries to the build, however I've noticed that the LIBS variable is cleared on row 166 in the nrf Makefile after the `USER_C_MODULES `have been included. Is there a reason why it is cleared? I want the nrf Makefile to be independent of my C module so I would prefer not to alter it myself in order to include my own libs.

**Comment by andrewleech** [comment]
I see what you mean in the Makefile; that really looks like a typo, a mistake. A PR to clean that up should be easy enough to get merged I'd expect!

**Comment by andrewleech** [comment]
That being said, in a user C module you can set lib / link flags more commonly with `LDFLAGS_USERMOD`

https://github.com/micropython/micropython/blob/master/examples%2Fusercmodule%2Fcppexample%2Fmicropython.mk#L12
